pages/Category.vue

```diff
@@ -223,6 +223,8 @@ import {
 
 const THEME_PAGE_SIZE = 12;
 const LAZY_LOADING_ACTIVATION_BREAKPOINT = 1024;
+var unsubscribeFromStoreAction = ''
+var aggs = ''
```

Comment:
Using global variables is not a good idea. It might cause race conditions in concurrent requests.

Reply:
@Fifciu 
The other solution can be to call the composeInitialPageState in the beforeRouteEnter hook. You can see the solution in this commit: https://github.com/vuestorefront/vsf-capybara/pull/560/commits/57cb6cdad7c7531d2feb4dc38e48028ebb926c49

Also, another thing that I can think of is to maintain a state for storing the aggregations and then using that state in our component, and thus we also not need to subscribe to the action.

Reply:
@Fifciu I have updated the solution to make use of state and getter to use the aggregations, please review.
https://github.com/vuestorefront/vue-storefront/pull/5392